server/lib/models/reaction.go

```diff
@@ -10,6 +14,58 @@ type Reaction struct {
 }
 
 func InitReaction() {
-	create_table(Reaction{}, "reactions", "Id")
+	t := create_table(Reaction{}, "reactions", "Id", "Persona")
+	t.AddIndex("created_locale", "INDEX", []string{"Created", "Locale"})
 }
 
+//create fixture
```

Comment:
__[gometalinter]__
* [golint] comment on exported type TopicInfo should be of the form "TopicInfo ..." (with optional leading article)